<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterEditorClass</class>
 <widget class="QWidget" name="RegisterEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RegisterEditor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="m_fControl">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="m_pbReset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="m_chbN">
        <property name="text">
         <string>N</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_chbZ">
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_chbV">
        <property name="text">
         <string>V</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_chbC">
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="m_fRegisters">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="m_fMode">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="m_rbHex">
        <property name="text">
         <string>HEX</string>
        </property>
        <attribute name="buttonGroup">
         <string>m_bgMode</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_rbDec">
        <property name="text">
         <string>DEC</string>
        </property>
        <attribute name="buttonGroup">
         <string>m_bgMode</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_rbOct">
        <property name="text">
         <string>OCT</string>
        </property>
        <attribute name="buttonGroup">
         <string>m_bgMode</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_rbBin">
        <property name="text">
         <string>BIN</string>
        </property>
        <attribute name="buttonGroup">
         <string>m_bgMode</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>m_pbReset</sender>
   <signal>clicked()</signal>
   <receiver>RegisterEditorClass</receiver>
   <slot>resetRegisters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>606</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>resetRegisters()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="m_bgMode"/>
 </buttongroups>
</ui>
